### PR TITLE
Add EntityStats and Vanish commands

### PR DESCRIPTION
**Pull request by d1gress**

### CODE REVIEW

**Comment by S-B99** [comment]
Next time I'd prefer seperate prs as branches but thanks, looks good

File: src/main/java/me/zeroeightsix/kami/command/commands/VanishCommand.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import net.minecraft.entity.Entity;
+
+/**
+ * Created by d1gress/Qther on 25/11/2017.
+ */
+
+public class VanishCommand extends Command {
+
+    private static Entity vehicle;
+    Minecraft mc = Minecraft.getMinecraft();
```
Comment (S-B99) [inline src/main/java/me/zeroeightsix/kami/command/commands/VanishCommand.java]: Why is this needed?

Comment (Vonr) [reply src/main/java/me/zeroeightsix/kami/command/commands/VanishCommand.java]: wdym

Comment (S-B99) [reply src/main/java/me/zeroeightsix/kami/command/commands/VanishCommand.java]: The mc =

You don't have to declare it?

Comment (Vonr) [reply src/main/java/me/zeroeightsix/kami/command/commands/VanishCommand.java]: easier for me to port cos bleachhack predefines it in Command or something along the lines of that

Comment (S-B99) [reply src/main/java/me/zeroeightsix/kami/command/commands/VanishCommand.java]: Nevermind, you're right yea

Comment (Vonr) [reply src/main/java/me/zeroeightsix/kami/command/commands/VanishCommand.java]: its very not needed so feel free to change it
